src/content/administration/dashboard.mdx

```diff
@@ -177,12 +177,64 @@ From this page you can follow a simple step-by-step guide to get the major compo
+
+In that way, if you want to define static credentials (username and password) for a private registry, you just need to create a secret in the Okteto namespace with the user and pass to be used. After that, you have to define a CRD with some information. The name of the resource has to be the hostname of the registry, so imagine you want to add a configuration for docker's registry, you will need to create a resource like this:
+
+```yaml
+apiVersion: admin.okteto.com/v1
+kind: Registry
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: okteto
+  name: index.docker.io # hostname of the registry to be configured
+  namespace: okteto
+spec:
+  static:
+    passwordSecretRef:
+      key: <field in referenced secret> # key of the secret which contains the password
+      name: <secret-name> # name of the secret where the password is stored
+    userSecretRef:
+      key: <field in referenced secret> # key of the secret which contains the username
+      name: <secret-name> # name of the secret where the username is stored
+```
+
+In the case of AWS, where you have to specify the Access Key and Access Secret, the process is similar to the static credentials one. You will need to create a secret with those values, and then, create a CRD like this (the name of the resource has to be the hostname of the registry to be configured and it has to end with `.amazonaws.com`):
+
+```yaml
+apiVersion: admin.okteto.com/v1
+kind: Registry
+metadata:
+  labels:
+    app.kubernetes.io/managed-by: okteto
+  name: 111122223333.dkr.ecr.eu-central-1.amazonaws.com # hostname of the registry to be configured
+  namespace: okteto
+spec:
+  awsIamUser:
+    accessKeyIDSecretRef:
+      key: <field in referenced secret> # key of the secret which contains the access key
+      name: <secret-name> # name of the secret where the access key is stored
+    secretAccessKeySecretRef:
+      key: <field in referenced secret> # key of the secret which contains the access secret key
+      name: <secret-name> # name of the secret where the access secret is stored
+```
 
-<p align="center"><Image src="/docs/administration/images/registry_credentials_detail.png" alt="Registry credentials detail" width="700" /></p>
\ No newline at end of file
+Last thing worth to mention is related to the label `app.kubernetes.io/managed-by: okteto` in the CRDs. Adding that label means that the credentials will be editable from the UI, so any admin could come to this view and edit them. If the lifecycle of those credentials is managed directly by you outside Okteto, you can remove that label. Okteto will keep considering those credentials to be used, and they will keep showing in this view but as read only.
```

Comment:
I think this is always the case when using CRDs. I would remove the label `app.kubernetes.io/managed-by: okteto` from the samples and its explanation

Reply:
+1. I don't think we should encourage the use of this label to resources created directly via `kubectl`. This could bring problems. With this label enabled (since it's managed by us) we assume a single secret with "username" and "password" keys and a predefined secret name. If they use this label and use different keys or a different secret name it will work initially but it will be editable through the UI and once edited through the UI it will create a new secret.

To recap, if they want to add the label they need to:

1. Use the same secret for user/password
2. The secret keys **must** be `username` and `password`
3. The secret name must be: `{host}-regcreds`

Reply:
Ok, agree to explain it in in the other way, but I would refer somehow to that label or explain it. If someone wants to migrate their current registries defined here and migrated automatically by Okteto to their git ops approach, they would have to remove that label or it will have problems if change the value directly from the UI

Reply:
@maroshii Related to this:

```
To recap, if they want to add the label they need to:

* The secret keys must be username and password
* The secret name must be: {host}-regcreds
```

Aren't we reusing the secret and field names defined in the CRD when updating or managing the credentials?

Reply:
Yes we are and the read path should not break but when we create/update we make some assumptions. For eg, we assume they are stored in a single secret and we only read `reg.Spec.Static.UserSecretRef.Name`.

The whole point of that label was to be able to make those assumptions and make change without affecting users. I think we should keep that property and not expose these internals.

Reply:
Ok, thanks for the explanation. I moved all this to another section. It's own page and removed the reference for the label. Take a look when you have a chance